blog/2024-06-06-we-are-on-gpt-4o.mdx

```diff
@@ -0,0 +1,27 @@
+---
+slug: we-are-on-chatgpt-4o
+title: We are on GPT 4o 🚀
+description: Tolgee Translator is now backed by GPT-4o, the latest OpenAI's flagship model.
+authors: [jcizmar]
+tags: [chatgpt, tolgee, ai, tolgee-translator]
+---
+
+Great news! 🚀 Tolgee Translator is now backed by GPT-4o, the latest OpenAI's flagship model. It returns better results, and it is much less error-prone.
```

Comment:
Here’s a closer look at what this means for you.

Reply:
What does it mean?

Reply:
oh, sorry. I'd just put this text at the end of your sentence